integration/basic_test.go

```diff
@@ -550,6 +550,100 @@ func TestDebugError(t *testing.T) {
 	})
 }
 
+func TestDiffDebugError(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+
+	for name, tc := range map[string]struct {
+		input string
+
+		err *mongo.CommandError
+		res bson.D
+	}{
+		"Empty": {
+			input: "",
+			err: &mongo.CommandError{
+				Code:    1,
+				Name:    "InternalError",
+				Message: "command failed",
+			},
+		},
+		"Ok": {
+			input: "ok",
+			res:   bson.D{{"ok", float64(1)}},
+		},
+		"CodeZero": {
+			input: "0",
+			err: &mongo.CommandError{
+				Code:    1,
+				Name:    "InternalError",
+				Message: "Unset",
+			},
+		},
+		"CodeOne": {
+			input: "1",
+			err: &mongo.CommandError{
+				Code:    1,
+				Name:    "InternalError",
+				Message: "InternalError",
+			},
+		},
+		"CodeNotLabeled": {
+			input: "33333",
+			err: &mongo.CommandError{
+				Code:    1,
+				Name:    "InternalError",
+				Message: "ErrorCode(33333)",
+			},
+		},
+		// TODO https://github.com/FerretDB/FerretDB/issues/3672
```

Comment:
Created a new issue in FerretDB because issue link for dance fails on todo linter check :see_no_evil: